bukuserver/templates/bukuserver/lib.html

```diff
@@ -31,17 +32,37 @@
   {% endif %}
 {% endmacro %}
 
-{% macro fetch_checkbox(checked=True) %}
+{% macro fetch_checkbox(checked=True, modal=False) %}
```

Comment:
`modal=` is used for placing the Fetch checkbox within the buttons row in modals (as per the screenshots)